content/builds/power-deadeye.md

```diff
@@ -0,0 +1,88 @@
+3. For the dagger rotation, which is mostly autos, use {{ skill(id="16432") }} for stealth and queue {{ skill(id="13005") }} to deal heavy damage. Ensure that you are not 'revealed' when you do the next stealth and backstab cycle.
+
+## Sustainability
+Your heal skill {{ skill(id="13050") }} should provide enough heals per hit. You can opt to take {{ trait(id="1702") }} but this is not an efficient way for sustainability.
+Dagger provides an additional evade on skill 3.
+
+## Crowd Control
+You have decent CC by just using {{ skill(id="13012") }} (Pistol 4). You can take {{ skill(id="13132") }} (elite) for a minimal DPS loss if you still need more CC. 
+
+This build may underperform at high end raiding but should put you at the top in PUGs. The build can however do extremely well in fights where ranged pierce is able to double your DPS. Some targets are tall and piercing is not automatic and your rounds may fire upwards, missing all posterior targets.
+
+You need to manually select the furthest target in order to hit as many targets as possible. Ranged pierce can also be done with a rifle with better DPS but the reduced mobility and increased difficulty in execution may not be feasible in certain fights. If using {{ skill(id="41158") }}, it is recommended to mark the closer target and then select the further target. A screenshot is provided at the end of the benchmark video for reference. 
+
+A few examples where this advantage can be used are:
+1.  Kaineng Overlook - both split phases
+2.  Harvest Temple - Any phase with adds
+3.  Old Lion’s Court - Any phase with more than 1 boss
+4.  W1 Gorseval - split phase
+5.  W2 Slothasor and his slublings
+6.  W3 Escort Tower Thief
+
+If taking practiced tolerance and daggers as an off-hand set, your daggers are your cleave options as taking practiced tolerance removes the cleave on pistol.
+
+## Niche Roles
+1.  W1 Sabetha - This build is ideal to do mechanics as you can carry a shadowstep and overcap boons, sacrificing very little for doing cannons.
+2.  W3 Xera - This build is able to function at range decently without stacking, making it ideal to step on buttons during the 3-button split phase
+3.  W6 Qadim - While solo lamp is ideally performed using daredevil (same equipment), you may choose to use this build instead if 2 players are sent to do Lamp. 
+4.  W7 QtP - This build is able to pylon kite. A video is provided for reference.
+
+## Videos
+I have benched the build with a slight increase in hp by using mixed ascended gear without infusions and budget food. The reason for increasing the vitality is that the intensity of the build may be lowered and DPS uptime can be increased in certain instances. e.g. You should be able to take the sniper shot on Kaineng Overlook NM without dodging.
+
+DPS benchmark (28-29k)
```

Comment:
If you make these headers (with #), then they will show up as clickable headings in the sidebar

Reply:
I have removed some of the headers and classified subheaders using **subheader** instead